testing/data/DATA_LICENSES.md

```diff
@@ -28,6 +28,7 @@
 - duck_invalid.gltf: Copyright 2006 Sony Computer Entertainment Inc.: SCEA Shared Source License
 - flame_mixed.mdl: LibreQuake Project: BSD-3-Clause
 - HeadMRVolume\*: VTK Data: BSD-3-Clause
+- hornedlizard_small_d\*.spz: Modified from SPZ samples: MIT
```

Comment:
whats the copyright on it ?

Reply:
added "Niantic" and the link to the repo.